docs/tutorials/notebooks/icnn_inits.ipynb

```diff
@@ -300,7 +136,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Next, we define the architectures parameterizing the dual potentials $f$ and $g$. These need to be parameterized by ICNNs. You can adapt the size of the ICNNs by passing a sequence containing hidden layer sizes. While ICNNs are by default containing partially positive weights, we can solve the problem using approximations to this positivity constraint (via weight clipping and a weight penalization). For this, set `positive_weights` to `True` in both the {class}`~ott.solvers.nn.icnn.ICNN` architecture and {class}`~ott.solvers.nn.neuraldual.NeuralDualSolver` configuration. For more details on how to customize the ICNN architectures, we refer you to the documentation."
+    "Next, we define the architectures parameterizing the dual potentials $f$ and $g$. These need to be parameterized by ICNNs. You can adapt the size of the ICNNs by passing a sequence containing hidden layer sizes. While ICNNs are by default containing partially positive weights, we can solve the problem using approximations to this positivity constraint (via weight clipping and a weight penalization). For this, set `positive_weights` to `True` in both the {class}`~ott.solvers.nn.models.ICNN` architecture and {class}`~ott.solvers.nn.neuraldual.W2NeuralDual` configuration. For more details on how to customize the ICNN architectures, we refer you to the documentation."
```

Comment:
In `ICNN`, the argument is `pos_weights`, not `positive_weights`, can you please update the notebook?